apps/hosts/utils.py

```diff
@@ -15,4 +15,8 @@ def is_user_a_host_of_challenge(user, challenge_pk):
     return Challenge.objects.filter(pk=challenge_pk, creator_id__in=challenge_host_teams).exists()
 
 
+def is_user_part_of_host_team(user, host_team):
```

Comment:
This is to make sure only the creator of the host team can add participants to it, yes?

Reply:
Yeah this added to check if any person is part of the host team or not and hence based on the result returned from this, the person will be allowed to add people to his team.